toolkit/scripts/check_static_glibc.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/python3
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+from pyrpm.spec import Spec
+from os.path import dirname, realpath
+
+import argparse
+import re
+import sys
+
+def get_glibc_version():
+    glibc_spec = Spec.from_file("SPECS/glibc/glibc.spec")
+    epoch_prefix = "{}:".format(glibc_spec.epoch) if glibc_spec.epoch else ""
+    release_without_dist = glibc_spec.release.replace("%{?dist}", "")
+    release_suffix = "-{}".format(release_without_dist)
+    glibc_version = epoch_prefix + glibc_spec.version + release_suffix
+    return glibc_version
+
+def check_spec(path, glibc_version):
+    spec = Spec.from_file(path)
+    for br in spec.build_requires:
+        if br.name == "glibc-static":
```

Comment:
It might be worth to return an error in case there's no `BuildRequires: glibc-static...` in the spec. Or at least a warning, so that it's clear we're using this script wrong.

Reply:
The normal use case is to run this over _all_ spec files, we'd get ~2000 warnings each run so I thought it better to be quiet (channelling that "if nothing's wrong be silent" adage that many linux tools use)

Reply:
Well, as long as it's still quick.

Reply:
It takes ~1min to run over the whole spec set (`SPECS*/**/*.spec`)